docs/notebooks/introduction_demo.ipynb

```diff
@@ -376,7 +376,7 @@
     "from pzflow import Flow\n",
     "from tdastro import _TDASTRO_TEST_DATA_DIR\n",
     "\n",
-    "pzflow = Flow(file=_TDASTRO_TEST_DATA_DIR / \"test_flow.pkl\")\n",
+    "pzflow = Flow(file=\"../../tests/tdastro/data/test_flow.pkl\")\n",
```

Comment:
Why change the path? Is this needed for auto-notebook rendering?

Reply:
Yes - the path doesn't resolve the same way in CI and RTD as it does in unit tests, so the file isn't found.